example/lib/main.dart

```diff
@@ -0,0 +1,715 @@
+import 'dart:async';
+
+import 'package:example/search_country_repository.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/rx_form.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Widget Toolkit Demo',
+      theme: ThemeData.light().copyWith(
+          colorScheme: ColorScheme.fromSwatch(),
+          extensions: _widgetToolkitThemeExtensions(true)),
+      darkTheme: ThemeData.dark().copyWith(
+          colorScheme: ColorScheme.fromSwatch(),
+          extensions: _widgetToolkitThemeExtensions(false)),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+List<ThemeExtension> _widgetToolkitThemeExtensions(bool inLightMode) =>
+    inLightMode
+        ? [
+            WidgetToolkitTheme.light,
+            ItemPickerTheme.light,
+            SearchPickerTheme.light,
+            TextFieldDialogTheme.light,
+            EditAddressTheme.light,
+          ]
+        : [
+            WidgetToolkitTheme.dark,
+            ItemPickerTheme.dark,
+            SearchPickerTheme.dark,
+            TextFieldDialogTheme.dark,
+            EditAddressTheme.dark,
+          ];
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key});
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final boolStream = BehaviorSubject<bool>();
+
+  @override
+  Widget build(BuildContext context) {
+    final PageController controller = PageController();
+    return Scaffold(
+      appBar: AppBar(title: const Text('Widget Toolkit Example')),
+      resizeToAvoidBottomInset: true,
+      body: SafeArea(
+        child: PageView(
+          controller: controller,
+          children: <Widget>[
+            _buildShowcasePage(context,
```

Comment:
Avoid using helper method: watch this video for more details https://youtu.be/IOyq-eTRhvo